build.ps1

```diff
@@ -50,7 +50,7 @@ cmake `
 	-DLLVM_INCLUDE_TOOLS=ON `
 	-DLLVM_INCLUDE_UTILS=OFF `
 	-DLLVM_OPTIMIZED_TABLEGEN=ON `
-	-DLLVM_TARGETS_TO_BUILD="X86;AArch64" `
```

Comment:
[Quoting the documentation](https://www.llvm.org/docs/CMake.html):

> LLVM_TARGETS_TO_BUILD:STRING
>    Semicolon-separated list of targets to build, or _all_ for building all targets. Case-sensitive. Defaults to _all_. Example: -DLLVM_TARGETS_TO_BUILD="X86;PowerPC".

So without `LLVM_TARGETS_TO_BUILD`, it will build all possible targets, while we just want `X86` and `AArch64`. It's not foundamentally an issue to have more targets, except the time required to compile LLVM and the size of the artifact, but more importantly, I don't understand why selecting of subset of targets introduce an error.

Do you have an idea?

Reply:
With this option removed, rust build gives a different error which can be fixed with the other change proposed in https://github.com/wasmerio/wasmer/pull/2151